viewer/lib/pages/api_detail.dart

```diff
@@ -70,32 +71,38 @@ class ApiDetailPage extends StatelessWidget {
                 child: TabBarView(
                   children: [
                     ApiDetailCard(editable: true),
-                    CustomSplitView(
-                      viewMode: SplitViewMode.Horizontal,
-                      initialWeight: 0.33,
-                      view1: VersionListCard(),
-                      view2: VersionDetailCard(
-                        selflink: true,
-                        editable: true,
-                      ),
-                    ),
-                    CustomSplitView(
-                      viewMode: SplitViewMode.Horizontal,
-                      initialWeight: 0.33,
-                      view1: DeploymentListCard(),
-                      view2: DeploymentDetailCard(
-                        selflink: true,
-                        editable: true,
-                      ),
-                    ),
-                    CustomSplitView(
-                      viewMode: SplitViewMode.Horizontal,
-                      view1: ArtifactListCard(SelectionProvider.api),
-                      view2: ArtifactDetailCard(
-                        selflink: true,
-                        editable: true,
-                      ),
-                    ),
+                    narrow(context)
+                        ? VersionListCard()
+                        : CustomSplitView(
```

Comment:
Looking at this now, I want to revisit this and consider pushing the `narrow()` check inside `CustomSplitView`. I would do that now but want to first clean up #55 and merge that, since it has an important fix for a layout problem that was introduced with the tab views.